pkg/arvo/gen/sponsor.hoon

```diff
@@ -0,0 +1,6 @@
+::  Print the sponsor of this ship
+:-  %say
+
```

Comment:
Blank lines are against the style guide. Otherwise, looks great!

Reply:
Fixed this!